enable/gcbench/bench.py

```diff
@@ -122,9 +122,9 @@ def gen_timings(gc, func):
 
     times = np.array(times)
     return {
-        "max": times.max() * 1000,
-        "min": times.min() * 1000,
         "mean": times.mean() * 1000,
+        "min": times.min() * 1000,
+        "max": times.max() * 1000,
```

Comment:
I'm using the order of this dictionary when turning it into an HTML table

Reply:
should be fine now that we only support python >= 3.6.